src/test/field/getters.spec.ts

```diff
@@ -3,16 +3,25 @@ import { ObjectType, Field, compileObjectType } from '~/domains';
 
 describe('Fields based on getters', () => {
   it('Will work with getter field', async () => {
+    let foo: Foo;
     @ObjectType({ description: 'Simple product object type' })
     class Foo {
+      a: number;
       @Field()
       get bar(): number {
-        return 42;
```

Comment:
I don't think it's good way to perform tests by throwing errors to stop them.

There might be different ways to accomplish this like adding instance field and then returning result like `return this.foo` and that should be enough to test if instance was avaliable